site/content/integrate/apps/deployment/_index.md

```diff
@@ -3,9 +3,6 @@ title: "Deployment"
 heading: "Deploying Apps"
 description: "TODO"
 weight: 110
-_build:
- list: false
- render: false
```

Comment:
This change will show the link to this file in the documentation.  It was added to hide this page until the docs were complete.  

This is reverting the portion of [this PR](https://github.com/mattermost/mattermost-developer-documentation/pull/829/files) that hid this doc.